### ISSUE TITLE
CoFH mods got an update, mod refuses to load

### ISSUE THREAD

**Issue by bochen415** [CyberdyneCC/SimplyJetpacks-2#75]
As title
![](https://i.imgur.com/SZxdAnZ.png)
My version: SimplyJetpacks2-1.12.2-2.2.2.30

**Comment by ghost** [comment]
I have this same prob so I cant update my pack till this is fixed

**Comment by Bulldog83** [comment]
Same with forge server:
```
[21:39:40] [Server thread/FATAL] [FML]: net.minecraftforge.fml.common.MissingModsException: Mod simplyjetpacks (Simply Jetpacks 2) requires [thermalexpansion@[5.3.6,5.4.0)]
[21:39:40] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraftforge.fml.common.MissingModsException: Mod simplyjetpacks (Simply Jetpacks 2) requires [thermalexpansion@[5.3.6,5.4.0)]
	at net.minecraftforge.fml.common.Loader.sortModList(Loader.java:275) ~[Loader.class:?]
	at net.minecraftforge.fml.common.Loader.loadMods(Loader.java:580) ~[Loader.class:?]
	at net.minecraftforge.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:97) ~[FMLServerHandler.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:332) ~[FMLCommonHandler.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:125) ~[nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_171]
```

**Comment by MokahTGS** [comment]
Yup, disabling Jetpacks in our modpack until this is fixed.

**Comment by MasterBuilder747** [comment]
what i dont understand is why it cannot be above 5.4, since when does a mod not like a newer version of its dependent mod?

**Comment by Tomson124** [comment]
The thing is the devs of CoFH mods said they would some bigger backend changes with 5.4.x so I limited it to 5.3.x so nothing breaks in bad ways (like vanishing items or things like it).
But as I had some trouble in my personal life (breakup, ill and some other things...) recently I did not know they were that close to version 5.4 which is why it doesn't work at the moment. Will fix that (hopefully) todays evening (Germany) or at the latest tomorrow.
